hack/run-e2e-kind.sh

```diff
@@ -31,6 +31,8 @@ export CLUSTER_CONTEXT="--name ${CLUSTER_NAME}"
 
 export KIND_OPT=${KIND_OPT:="--config ${VK_ROOT}/hack/e2e-kind-config.yaml"}
 
+NAMESPACE="volcano-system"
```

Comment:
How about look for env NAMESPACE first, if not exists then set a defaule value?

Reply:
ok, i will modify it based on the Parameter Expansion feature of Bash
```bash
NAMESPACE=${NAMESPACE:-kube-system}
```
and i want to modify line 26-28 at the same time
```bash
CLUSTER_NAME=${CLUSTER_NAME:-integration}
```